index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+/* eslint-disable no-console */
+
+class Executor {
+    /**
+     * Constructor for Executor
+     * @method constructor
+     * @param  {Object}    config Configuration
+     * @return {Executor}
+     */
+    constructor(config) {
+        this.config = config;
+    }
+
+    /**
+     * Start the executor
+     * @method start
+     */
+    start() {
+        console.error('start is not implemented');
```

Comment:
well the reason petey used console in the datastore-base was that he wanted the base class as small as possible and not bring in any external dependencies